### PR TITLE
JENKINS-45719 allow disabling of run description

### PR DESCRIPTION
**Pull request by i386**
# Description

Adds a system property to disable run descriptions

See [JENKINS-45719](https://issues.jenkins-ci.org/browse/JENKINS-45719).

# Submitter checklist
- [x] Link to JIRA ticket in description, if appropriate.
- [x] Change is code complete and matches issue description
- [x] Appropriate unit or acceptance tests or explanation to why this change has no tests
- [x] Reviewer's manual test instructions provided in PR description. See Reviewer's first task below.

# Reviewer checklist
- [ ] Run the changes and verified the change matches the issue description
- [ ] Reviewed the code
- [ ] Verified that the appropriate tests have been written or valid explanation given

### CODE REVIEW

File: blueocean-pipeline-api-impl/src/main/java/io/jenkins/blueocean/rest/impl/pipeline/PipelineRunImpl.java [hunk 1]
```diff
@@ -59,11 +59,6 @@ public PipelineRunImpl(WorkflowRun run, Reachable parent, BlueOrganization organ
         super(run, parent, organization);
     }
 
-    @Exported(name = "description")
```
Comment (i386) [inline blueocean-pipeline-api-impl/src/main/java/io/jenkins/blueocean/rest/impl/pipeline/PipelineRunImpl.java]: Declared on AbstractRunImpl already

**Comment by michaelneale** [comment]
unlikely to pass, but if it does - LGTM

**Comment by i386** [comment]
@vivek we could do that but this is just for an internal customer that wishes to turn it off for the moment.

**Comment by vivek** [comment]
@i386 Hope they don't come back asking can we enable it for other descriptions that blueocean can display. Anyways change looks fine so its a 🐝 but I think a proper fix would save us effort cleaning this code in future.

**Comment by michaelneale** [comment]
update to master

**Comment by michaelneale** [comment]
:bee:

**Pull request merged**